docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md

```diff
@@ -34,12 +34,12 @@ If you choose this option, you can change the configuration to match your deploy
    ```bash
    git clone https://github.com/grafana/mimir.git
    ```
-2. Install dependencies
+2. Review the mixin configuration at `operations/mimir-mixin/config.libsonnet` and apply your changes if required
```

Comment:
```suggestion
2. Review the mixin configuration at `operations/mimir-mixin/config.libsonnet`, and apply your changes if necessary.
```

Reply:
This is a preference to match voice and tone, and to reduce the complexity of past participles.